com.unity.render-pipelines.core/Editor/LookDev/LookDev.cs

```diff
@@ -43,6 +49,16 @@ internal static Context currentContext
             private set => s_CurrentContext = value;
         }
 
+        static Context TryFindCurrentContext()
+        {
+            Context context = s_CurrentContext;
+
+            if (context != null)
+                return context;
+
+            return Resources.FindObjectsOfTypeAll<Context>().FirstOrDefault();
```

Comment:
Can we be sure that we not load a context that should be garbage collected ?

Reply:
I'm not sure to see why it matters?

In any case, the static field is cleared after the domain reload, if the ScriptableObject survived the domain reload, then I expect it to never be garbage collected.